Mathlib/Combinatorics/SimpleGraph/Finite.lean

```diff
@@ -219,6 +219,10 @@ theorem card_neighborSet_eq_degree : Fintype.card (G.neighborSet v) = G.degree v
 theorem degree_pos_iff_exists_adj : 0 < G.degree v ↔ ∃ w, G.Adj v w := by
   simp only [degree, card_pos, Finset.Nonempty, mem_neighborFinset]
 
+theorem degree_eq_zero_iff_not_mem_support : G.degree v = 0 ↔ v ∉ G.support := by
```

Comment:
Can you also add a version saying the degree is positive if and only if v is in the support